### DISCUSSION TITLE
How to increase zoom range?

### DISCUSSION THREAD

**Discussion by Nodding** [AlexandreRouma/SDRPlusPlus#577] [Q&A] [answered]
I am new to SDR in general and was wondering why the zoom is locked to a max range of about 3MHz. I would want to be able to see as much as possible if possible. It seems locked to this range on other software too, such as CubicSDR, so I'm unsure if this is just an unspoken standard amongst this kind of software.

**Comment by the-gobleo-tinkerer** [comment, marked as answer]
The "max zoom range" is determined by the sample rate of your SDR hardware, not the software. With an RTL-SDR the maximum sample rate is 3.2M samples per second, hence only 3.2MHz of frequencies at one time. SDRs with higher sample rates will allow you to view a wider range: for example an Airspy Mini with a 6M sample rate will give you at most 6MHz of simultaneous spectrum display.

To confirm this, change the sample rate in SDR++'s "Source" tab to some other value and you should see your "max range" change accordingly.

To see a wider range "at the same time" with the same RTL-SDR you need software like [xmikos/qspectrumanalyzer](https://github.com/xmikos/qspectrumanalyzer/) and [pavels/spektrum](https://github.com/pavels/spektrum) that shifts this 3.2MHz window to scan the freq range for you. However you won't be able to decode / demodulate anything (I'm not aware of any software that can), since each 3.2MHz slice will only be sampled intermittently.

**Reply by Nodding** [reply]
Great, that clears it up perfectly! Thanks!
